src/main/java/kotlinx/reflect/lite/impl/KClasses.kt

```diff
@@ -0,0 +1,15 @@
+@file:JvmName("KClasses")
+
+package kotlinx.reflect.lite.impl
+
+import kotlinx.reflect.lite.*
+import kotlinx.reflect.lite.descriptors.*
+
+/**
+ * Returns the primary constructor of this class, or `null` if this class has no primary constructor.
+ * See the [Kotlin language documentation](http://kotlinlang.org/docs/reference/classes.html#constructors)
+ * for more information.
+ */
+@SinceKotlin("1.1")
+val <T : Any> KClass<T>.primaryConstructor: KFunction<T>?
```

Comment:
Is it being tested anywhere?

Reply:
No, it wasn't tested then, now I've fixed the implementation and added a `tests.constructors.primaryConstructor` test

Reply:
Hopefully, every other signature is covered :)

Eventually, I expect #24 to help with that